src/Windowing/Silk.NET.Windowing.Glfw/GlfwWindow.cs

```diff
@@ -521,7 +522,21 @@ public override void Dispose()
             GC.SuppressFinalize(this);
         }
 
-        public nint GetProcAddress(string proc, int? slot = default) => _glfw.GetProcAddress(proc);
+        public nint GetProcAddress(string proc, int? slot = default)
+        {
+            var ret = _glfw.GetProcAddress(proc);
+            GLFW.Glfw.ThrowExceptions();
+            if (ret == 0)
+            {
+                Throw(proc);
+            }
+
+            return ret;
+            static void Throw(string proc) => throw new SymbolLoadingException(proc);
+        }
+
+        public bool TryGetProcAddress(string proc, out nint addr, int? slot = default)
+            => (addr = _glfw.GetProcAddress(proc)) != 0;
```

Comment:
Not an expert, but doesn't GLFW.ThrowExceptions(); throw a managed exception, so this could still throw an exception?

Reply:
GLFW.ThrowExceptions isn't called here, but on Windows this effectively implicitly called so the point still stands, will fix.